modules/translations/pages/index.adoc

```diff
@@ -16,6 +16,72 @@ The texts provided are not final and are subject to change. In particular,
 languages other than English have often been obtained via automated translation, and have not been reviewed yet. We will continue to add and update procurement labels in every future release of the SDK.
 ====
 
+[#index]
+== Index file
```

Comment:
The term "index file" probably does not mean much for readers.
Please see how the equivalent file is described in the docs page for codelists or view-templates.

Reply:
There is one use of index file in understanding-the-sdk.adoc, but in general we used "Information on xyz" or just a sentence like: "The file named view-templates.json provides information on ...".
Both are fine, I think I will try to use "Information on xyz" as it better separates the files from the information about the files. Inside of the text we can try to call it the index file (but not in the title as you said).